src/debugger/android/androidPlatform.ts

```diff
@@ -76,7 +83,7 @@ export class AndroidPlatform implements IAppPlatform {
     }
 
     private getPackageName(projectRoot: string): Q.Promise<string> {
-        return new Package(projectRoot).name().then(appName =>
+        return new Package(projectRoot, { fileSystem: this.fileSystem }).name().then(appName =>
```

Comment:
Why have this class know about how the package class accesses the file system rather than having giving this class a mockable way of creating a Package?